experimental/dds/tree2/src/feature-libraries/editable-tree/editableField.ts

```diff
@@ -392,11 +394,23 @@ export class FieldProxyTarget extends ProxyTarget<FieldAnchor> implements Editab
 		);
 	}
 
+	public getTreeStatus(): TreeStatus {
+		const path = this.cursor.getFieldPath().parent;
+		if (path === undefined) {
+			return TreeStatus.Deleted;
+		}
+		/**
+		 * TODO: This is a slow initial implementation which traverses the entire path up to the root of the tree.
+		 * This should eventually be optimized.
+		 */
```

Comment:
There is no reason to use TsDoc comments inside on a a line of code that is not declaring something. Just use // comments for cases like this.

Reply:
removed TsDoc comment and used //